src/nanorc/confdata/elisa_service.json

```diff
@@ -0,0 +1,3 @@
+{
+    "socket":"np04-srv-015:5005"
```

Comment:
More a note to self: This will probably need to be revised when we have the REST running